### PR TITLE
Feature/new about

### PR DESCRIPTION
**Pull request by cloudgge**

### CODE REVIEW

File: src/assets/js/navbar.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+console.log("called")
+const button = document.getElementById("collapse-button");
+const navbar = document.getElementById("collapseExample");
+console.log(button)
+console.log(navbar)
```
Comment (cbeyer42) [inline src/assets/js/navbar.js]:
```suggestion
const button = document.getElementById("collapse-button");
const navbar = document.getElementById("collapseExample");
```
Remove the debug logs.

File: README.md
Comment (cbeyer42) [inline README.md:1]: The unit test section can be removed.
The integration tests changed to cypress, so this section needs to be updated.

File: package.json [hunk 2]
```diff
@@ -3,51 +3,10 @@
...
-    "karma-chrome-launcher": "~2.1.1",
-    "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^1.2.1",
-    "karma-jasmine": "~1.1.0",
-    "karma-jasmine-html-reporter": "^0.2.2",
-    "protractor": "~5.1.2",
-    "ts-node": "~3.0.4",
-    "tslint": "~5.3.2",
-    "typescript": "~2.6.2"
+    "ces-theme": "https://github.com/cloudogu/ces-theme.git"
```
Comment (cbeyer42) [inline package.json]: use specific version of the ces-theme e.g.:
```suggestion
    "ces-theme": "https://github.com/cloudogu/ces-theme.git#v0.7.2"
```

File: src/assets/js/navbar.js (hunk 1 above)
Comment (cloudgge) [reply src/assets/js/navbar.js]: Done

File: README.md
Comment (cloudgge) [reply README.md:1]: Changed

File: package.json (hunk 2 above)
Comment (cloudgge) [reply package.json]: When trying to npm install, the install hangs without any indications as to why, this is also the case with verbose logs. Does it work on your end? I tried "@0.7.2" and "@v0.7.2".

**Review by cbeyer42 — APPROVED** [review]

**Pull request merged**